### PR TITLE
UICIRC-856 - Add metadata info to view of Patron Notice Templates (Settings > Circulation > Patron Notice Templates)

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose
Add metadata info to view of Patron Notice Templates (Settings > Circulation > Patron Notice Templates)

## Refs
https://issues.folio.org/browse/UICIRC-856

## Screenshots
0bKFBbcKpb

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/settings/PatronNotices/PatronNoticeDetail.js [hunk 1]
```diff
@@ -1,173 +1,73 @@
...
-    );
-  }
-}
+          { emailTemplate &&
+          <PatronNoticeEmailSection
+            notice={notice}
+            locale={locale}
+            emailTemplate={emailTemplate}
+          />
+            }
```
Comment (mariia-aloshyna) [inline src/settings/PatronNotices/PatronNoticeDetail.js:60]: It looks like something happened with the indents

File: src/settings/PatronNotices/PatronNoticeDetail.test.js [hunk 2]
```diff
@@ -2,197 +2,113 @@ import React from 'react';
...
-  PreviewModal,
-  tokensReducer,
-} from '@folio/stripes-template-editor';
+import buildStripes from '../../../test/jest/__mock__/stripes.mock';
 
+import { Metadata } from '../components';
 import PatronNoticeDetail from './PatronNoticeDetail';
+import {
+  PatronNoticeAboutSection,
+  PatronNoticeEmailSection
```
Comment (mariia-aloshyna) [inline src/settings/PatronNotices/PatronNoticeDetail.test.js:20]:
```suggestion
  PatronNoticeEmailSection,
```

File: src/settings/PatronNotices/components/ViewSections/PatronNoticeAboutSection/PatronNoticeAboutSection.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Col,
+  KeyValue,
+  Row
```
Comment (mariia-aloshyna) [inline src/settings/PatronNotices/components/ViewSections/PatronNoticeAboutSection/PatronNoticeAboutSection.js:7]:
```suggestion
  Row,
```

File: src/settings/PatronNotices/components/ViewSections/PatronNoticeAboutSection/PatronNoticeAboutSection.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+} from '@folio/stripes/components';
+
+import { FormattedMessage } from 'react-intl';
```
Comment (mariia-aloshyna) [inline src/settings/PatronNotices/components/ViewSections/PatronNoticeAboutSection/PatronNoticeAboutSection.js:10]: Shouldn't this be listed right after react imports and before stripes?

File: src/settings/PatronNotices/components/ViewSections/PatronNoticeEmailSection/PatronNoticeEmailSection.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        </Col>
+      </Row>
+      <PreviewModal
+        open={openPreview}
+        header={
+          <FormattedMessage
+            id="ui-circulation.settings.patronNotices.view.previewHeader"
+            values={{ name: notice.name }}
+          />
+          }
```
Comment (mariia-aloshyna) [inline src/settings/PatronNotices/components/ViewSections/PatronNoticeEmailSection/PatronNoticeEmailSection.js:70]:
```suggestion
        }
```

**Review by mariia-aloshyna — APPROVED** [review]

File: src/settings/PatronNotices/components/ViewSections/PatronNoticeEmailSection/PatronNoticeEmailSection.test.js [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+describe('PatronNoticeEmailSection', () => {
+  const testNotice = {
+    name: 'testName',
+    desciption: 'testDescription',
+    category: 'testCategory',
+    localizedTemplates: {
+      en: {
+        header: 'header',
+      },
+    }
```
Comment (artem-blazhko) [inline src/settings/PatronNotices/components/ViewSections/PatronNoticeEmailSection/PatronNoticeEmailSection.test.js:54]:
```suggestion
    },
```

File: src/settings/PatronNotices/components/ViewSections/PatronNoticeEmailSection/PatronNoticeEmailSection.test.js [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+    );
+  });
+
+  describe('open/close preview dialog', () => {
+    const openPreviewDialog = () => {
+      const previewButton = screen.getByRole('button', { name: /preview/i });
+      fireEvent.click(previewButton);
+    };
+    const closePreviewDialog = () => {
+      const previewModal = screen.getByTestId('previewModal');
```
Comment (artem-blazhko) [inline src/settings/PatronNotices/components/ViewSections/PatronNoticeEmailSection/PatronNoticeEmailSection.test.js:141]: Could you add `'previewModal'` to the object with ids?

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**